.github/workflows/deploy-preview.yml

```
@@ -0,0 +1,19 @@
+name: Deploy to Preview Channel
+
+on:
+  pull_request:
```

```suggestion
  - pull_request
```

でも、と思いましたが些細なので良いかと〜

---

あーありがとうございます、こり詳細指定の名残なので変えちゃいますね！